docs/explanation/parts.rst

```diff
@@ -0,0 +1,263 @@
+Parts
+=====
+
+In the Craft Parts framework, parts are descriptions of the components to be
+build for deployment, either individually or as part of a larger build
+containing many components.
+
+Tools like `Snapcraft`_ and `Juju`_ that use the concepts of parts to
+describe a build process typically accept specifications of parts in YAML format. This allows each part to be described in a convenient,
+mostly-declarative format.
+
+When the Craft Parts framework is used to process a part on behalf of a tool
```

Comment:
for clarity and completeness, i would bullet point the steps and briefly explain them, the first three are, but the last three are not - worth also emphasizing what build means (e.g.: compilation), and carefully explain the difference between stage and prime; also might be worth explaining what the overlay onto a base filesystem actually means (not a full blown description just a sentence or two)

Reply:
I wasn't sure I should say too much about the steps at the start. The problem is that you need to know that there are steps because the part contains entries for things that occur in different steps.
I've expanded the sentence to be a list and added some more information.